### PR TITLE
custom workspace example 

### PR DESCRIPTION
**Pull request by fdurchdewald**

### CODE REVIEW

File: ur_documentation/docs/source/examples/my_robot_cell/start_ur_driver.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
+=========================
+Start the ur_robot_driver
+=========================
+
+Now that everything is in place, we can proceed to start up a driver instance. To initialize the ``ur_robot_driver`` using our customized workspace description, we need to create a launch file.
```
Comment (fmauch) [inline ur_documentation/docs/source/examples/my_robot_cell/start_ur_driver.rst:5]:
```suggestion
Now that everything is in place, we can proceed to start up a driver instance. To initialize the ``ur_robot_driver`` using our customized workcell description, we need to create a launch file.
```

File: ur_documentation/docs/source/examples/my_robot_cell/start_ur_driver.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+Remember that we haven't actually invoked the controller spawner node yet, so be sure to call the function for the ROS2 controllers you want to use.
+
+We can start the driver by running:
+
+.. code-block:: bash
+
+    #start the driver 
+    ros2 launch my_robot_cell_control start_robot.launch.py ur_type:=ur20 robot_ip:=yyy.yyy.yyy.yyy
```
Comment (fmauch) [inline ur_documentation/docs/source/examples/my_robot_cell/start_ur_driver.rst:115]: Let's add a section where we start ursim and specify the IP address of the default URSim interface.

```suggestion
    ros2 launch my_robot_cell_control start_robot.launch.py ur_type:=ur20 robot_ip:=192.168.56.101
```

**Review by fmauch — CHANGES_REQUESTED** [review]
This looks like a very good tutorial so far. Please see my comments and suggestions.

File: my_robot_cell/my_robot_cell_description/meshes/notexture.png
[the remark this answers is not in the document]
Comment (fdurchdewald) [reply my_robot_cell/my_robot_cell_description/meshes/notexture.png:1]: notexture.png is used by 171005_Motek_Monitor_L.dae.

File: my_robot_cell/my_robot_cell_description/urdf/my_robot_cell.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    robot_ip="$(arg robot_ip)"
+    joint_limits_parameters_file="$(arg joint_limit_params)"
+    kinematics_parameters_file="$(arg kinematics_params)"
+    physical_parameters_file="$(arg physical_params)"
+    visual_parameters_file="$(arg visual_params)"
+    script_filename="$(arg ur_script_filename)"
+    output_recipe_filename="$(arg ur_output_recipe_filename)"
+    input_recipe_filename="$(arg ur_input_recipe_filename)"
+  >
+    <origin xyz="0 -0.05 0" rpy="0 0 ${pi}" />
```
[the remark this answers is not in the document]
Comment (fdurchdewald) [reply my_robot_cell/my_robot_cell_description/urdf/my_robot_cell.urdf.xacro:98]: It's actually not needed to view the robot description. However, the control package loads these files into the URDF, so we need to declare these arguments and pass them to the ur_robot macro.

File: my_robot_cell/my_robot_cell_moveit_config/launch/rsp.launch.py
[the remark this answers is not in the document]
Comment (fdurchdewald) [reply my_robot_cell/my_robot_cell_moveit_config/launch/rsp.launch.py:1]: When I wrote the tutorial, I intended to generate only the essential launch files. However, when I attempted to launch the move_group without rsp.launch.py, it didn't start. So, I decided to leave the launch files. Strangly, I just tried it again without those launch files, and it seems to work. I will remove them now.

File: ur_documentation/docs/source/examples/my_robot_cell/assemble_urdf.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Note that the **origin** argument is transmitted in a different manner than the other arguments.
+
+Before we can test our code, it's essential to build and source our Colcon workspace:
+
+.. code-block:: bash
+
+    #source and build your workspace
+    colcon build
+    source install/setup.bash
```
Comment (fmauch) [inline ur_documentation/docs/source/examples/my_robot_cell/assemble_urdf.rst:72]:
```suggestion
    # source and build your workspace
    # cd to your colcon workspace root
    colcon build
    source install/setup.bash
```

**Review by fmauch — APPROVED** [review]
We should think about actually restructuring this a bit, but apart from that I am rather happy with it now. Needs https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/977 before merging

**Pull request merged**